features/facebookImport/src/model/storage.js

```diff
@@ -62,6 +62,11 @@ export class ZipFile {
         return this.getContent(this._file.id);
     }
 
+    async stat(entry) {
+        const { polyOut } = this._pod;
+        return polyOut.stat(entry);
```

Comment:
Maybe check before that `entry` exists? It will propagate the error otherwise.

Reply:
`entry` comes from a file listing, so we should be pretty sure it exists. We did have some errors with this - paths from `readdir` weren't actually existing when we called `stat`. That's actually an error I'd _like_ to propagate, cause then the big error dialog shows up and people can report it :D Better than to silently swallow it, at this stage, I think.